README.md

```diff
@@ -0,0 +1,64 @@
+# cloud-provider-aws
+The AWS cloud provider provides the interface between a Kubernetes cluster and AWS service APIs. This project allows a Kubernetes cluster to provision, monitor and remove resources necessary for operation of the cluster.
+
+## AWS Labels
```

Comment:
As I mentioned in previous comments, I want to understand these flags a little better before telling users they are required.  I'd like explanation why they are required/what breaks without them.  We could potentially remove this section and create an issue to add it back in with appropriate explanation.